src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -205,6 +188,60 @@ public void TryGetConfig_ErrorsAreLogged()
             }
         }
 
+        [TestMethod]
```

Comment:
Is there a reason why these tests are here and not in FileConfigTests?

Reply:
They are all calling/testing  `CreateRequest` which is in `CfamilyHelper.cs`

Reply:
I understand, but they don't have to. The logic that was changed is in `FileConfig.TryGet`, so that's the method that should be tested.